components/expenses/ExpenseSummaryAdditionalInformation.js

```diff
@@ -123,8 +131,63 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
       flexDirection={['column', 'row']}
       alignItems={['stretch', 'flex-start']}
       flexWrap={['nowrap', 'wrap', null, 'nowrap']}
+      gridGap="12px"
     >
-      <PrivateInfoColumn data-cy="expense-summary-payee" borderless={borderless}>
+      {collective && (
+        <PrivateInfoColumn data-cy="expense-summary-collective">
+          <PrivateInfoColumnHeader>{formatCollectiveType(intl, collective.type)}</PrivateInfoColumnHeader>
+          <LinkCollective collective={collective}>
+            <Flex alignItems="center">
+              <Avatar collective={collective} radius={24} />
+              <Flex flexDirection="column" ml={2} mr={2} css={{ overflow: 'hidden' }}>
+                <Span color="black.800" fontSize="14px" fontWeight="700" truncateOverflow>
+                  {formatAccountName(collective.name, collective.legalName)}
+                </Span>
+                <Span color="black.900" fontSize="12px" truncateOverflow>
+                  @{collective.slug}
```

Comment:
Showing the handles is a small deviation in design (I informed them about it), but the handle is an essential detail for making sure it's the right profile; it makes it more difficult to impersonate someone